### PR TITLE
Time series plot dialog: added names_ptypes parameter to ensure labels are in correct order

### PR DESCRIPTION
**Pull request by dannyparsons**
@africanmathsinitiative/developers this is ready for review.

One minor change: `names_ptypes` added to `pivot_longer` to ensure that the reference and estimates columns always appear in the correct order in the labels of the graph.

This is the same as was done on the seasonality plot dialog as mentioned here https://github.com/africanmathsinitiative/R-Instat/pull/6006#issuecomment-705429670

### CODE REVIEW

**Comment by lloyddewit** [comment]
@shadrackkibet Please could you review? thanks

File: instat/dlgTimeSeriesPlot.vb [hunk 1]
```diff
@@ -53,10 +53,13 @@ Public Class dlgTimeSeriesPlot
...
     ' These functions construct the R code below. This stacks the two columns which is needed for plotting.
     ' df_stack <- df %>% 
-    '   pivot_longer(cols = c(gauge, estimate), names_to = "data", values_to = "value")
+    '   pivot_longer(cols = c(gauge, estimate), names_to = "data", values_to = "value", names_ptypes = list(data = factor(levels = c("gauge", "estimates"))))
     Private clsStackOperator As ROperator
     Private clsPivotLonger As RFunction
     Private clsPivotCFunction As RFunction
+    Private clsPivotListFunction As RFunction
+    Private clsPivotFactorFunction As RFunction
+    Private clsPivotFactorLevelsCFunction As New RFunction
```
Comment (shadrackkibet) [inline instat/dlgTimeSeriesPlot.vb:62]: Should this be `Private clsPivotFactorLevelsCFunction As RFunction` ?

**Review by shadrackkibet — CHANGES_REQUESTED** [review]

File: instat/dlgTimeSeriesPlot.vb (hunk 1 above)
Comment (dannyparsons) [reply instat/dlgTimeSeriesPlot.vb]: Thanks, done.

**Comment by shadrackkibet** [comment]
Looks good, over to @lloyddewit and @rdstern

**Review by lloyddewit — APPROVED** [review]

**Review by rdstern — APPROVED** [review]
Great - another one

**Pull request merged**